packages/rn-tester-e2e/tests/helpers/utils.js

```diff
@@ -8,40 +8,42 @@
  * @format
  */
 
-import {driver} from '../../jest.setup';
-
 type PlatformsReference = {
   ios: string,
   android: string,
 };
 
 class Utils {
   async checkElementExistence(locator: string): Promise<boolean> {
-    await driver.$(locator).waitForDisplayed();
-    return driver.$(locator).isDisplayed();
+    await $(locator).waitForDisplayed();
+    return $(locator).isDisplayed();
```

Comment:
You can just return `await $(locator).waitForDisplayed();`. Such as `return await $(locator).waitForDisplayed();`.

waitForDisplayed throws an error if it fails, or returns true if it succeeds. Adding `return $(locator).isDisplayed();` is unnecessary and will decrease perf by adding another API call.